pkg/ecdsa/tss/protocol_ready.go

```diff
@@ -86,6 +108,30 @@ func readyProtocol(
 
 	switch ctx.Err() {
 	case context.DeadlineExceeded:
+		for _, memberID := range group.groupMemberIDs {
```

Comment:
What about the current member? Will it be included in both `groupMemberIDs` and `readyMembers`?

Reply:
Correct. `groupMemberIDs` should contain identifiers of all group members and all members write to the broadcast channel.